vllm/model_executor/models/mamba.py

```diff
@@ -459,24 +441,15 @@ def sample(
     def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
         stacked_params_mapping = [
             # (param_name, shard_name, shard_id)
-            ("qkv_proj", "q_proj", "q"),
-            ("qkv_proj", "k_proj", "k"),
-            ("qkv_proj", "v_proj", "v"),
             ("gate_up_proj", "gate_proj", 0),
             ("gate_up_proj", "up_proj", 1),
```

Comment:
I don't see these modules in the checkpoint either https://huggingface.co/state-spaces/mamba-130m-hf?show_file_info=model.safetensors